src/component/tooltip/TooltipModel.ts

```diff
@@ -61,11 +61,18 @@ export interface TooltipOption extends CommonTooltipOption<TopLevelFormatterPara
     renderMode?: 'auto' | TooltipRenderMode   // TODO richText renamed canvas?
 
     /**
-     * If append popup dom to document.body
-     * Only available when renderMode is html
+     * @deprecated
+     * use appendTo: 'body' instead
      */
     appendToBody?: boolean
 
+    /**
+     * If append popup dom to some other dom element
+     * Only available when renderMode is html
+     */
+    appendTo?: ((chartContainer?: HTMLElement) => HTMLElement | undefined | null) | string | HTMLElement
```

Comment:
It seems the callback parameter `chartContainer` will never be undefined. Maybe the `?` can be removed.

Reply:
U r right. Gonna fix it later.